### DISCUSSION TITLE
ScriptReference/AudioSource.PlayDelayed

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#659] [Notes]
# ScriptReference/AudioSource.PlayDelayed

https://docs.unity3d.com/ScriptReference/AudioSource.PlayDelayed.html

**Comment by STARasGAMES** [comment]
This method doesn't work properly when Time.timeScale = 0. It plays sound without delay. 

It's safe to say that it does not repsect any timeScale rather than 1.
